engines/bastion_katello/app/assets/javascripts/bastion_katello/sync-plans/new/new-sync-plan.controller.js

```diff
@@ -44,7 +44,8 @@ angular.module('Bastion.sync-plans').controller('NewSyncPlanController',
         }
 
         $scope.createSyncPlan = function (syncPlan) {
-            var syncDate = new Date(syncPlan.startDate),
+            var syncDate = new Date(syncPlan.startDate.getTime() +
+                                    syncPlan.startDate.getTimezoneOffset() * 60000),
```

Comment:
You may want to create a variable (or better yet a `const`) for the 60,000 so it is immediately clear what it represents.